lib/pharos/cluster_manager.rb

```diff
@@ -74,9 +74,13 @@ def apply_phases
       end
 
       apply_phase(Phases::ConfigureSecretsEncryption, master_hosts, ssh: true, parallel: false)
+      apply_phase(Phases::SetupMaster, master_hosts, ssh: true, parallel: true)
+      apply_phase(Phases::UpgradeMaster, master_hosts, ssh: true, parallel: false)
+
+      apply_phase(Phases::MigrateWorker, config.worker_hosts, ssh: true, parallel: true)
+      apply_phase(Phases::ConfigureKubelet, config.hosts, ssh: true, parallel: true)
+
       apply_phase(Phases::ConfigureMaster, master_hosts, ssh: true, parallel: false)
-      apply_phase(Phases::MigrateWorker, config.worker_hosts, ssh: true, parallel: true, master: master_hosts.first)
```

Comment:
The `master: ...` was already removed as part of #302, but snuck back in during the #293 merge.